### DISCUSSION TITLE
Override an abstracted utility classes

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [tailwindlabs/tailwindcss#4046] [Help] [answered]
```CSS
.btn {
  @apply p-1 font-semibold  ...;
}
```
```javascript
// ...
<button class="btn px-6">Button<button>
```
I am trying to override one of the abstracted utility classes `p-1` but I could not, how can I achieve that?

**Comment by adamwathan** [comment, marked as answer]
Make sure the btn class is defined within the "components" layer and you should be good:

https://tailwindcss.com/docs/extracting-components#extracting-component-classes-with-apply
